manifests/repo/redhat.pp

```diff
@@ -13,6 +13,7 @@
     /(?i:Amazon)/ => '6',
     default       => '$releasever',  # Yum var
   }
+  $version = $yum_repo.match('.\d$')[0]
```

Comment:
`.\d$` feels weird: any char followed by a digit at the end of a line.

I guess what you mean here is any number of digits at the end of the string? `\d+\z`